packages/xdl/src/start/ExpoUpdatesManifestHandler.ts

```diff
@@ -96,19 +171,31 @@ export async function getManifestResponseAsync({
       key: mainModuleName,
       contentType: 'application/javascript',
       url: bundleUrl,
+      fileExtension: '.fake', // shouldn't need this but android crashes without it
     },
     assets,
     metadata: {}, // required for the client to detect that this is an expo-updates manifest
     extra: {
-      eas: {}, // TODO(wschurman): somehow inject EAS config in here if known
+      eas: {
+        projectId: projectId ?? undefined,
+      },
       expoClient: {
         ...expoConfig,
         hostUri,
       },
       expoGo: expoGoConfig,
+      scopeKey,
     },
   };
 
+  if (acceptSignature) {
+    const shouldUseAnonymousManifest = await shouldUseAnonymousManifestAsync();
+    const manifestSignature = shouldUseAnonymousManifest
+      ? 'UNSIGNED'
```

Comment:
Do we need this (as opposed to just omitting the header)?

Reply:
Probably not. The client currently looks for it but treats it the same as omitting it it seems: https://github.com/expo/expo/blob/master/packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.java#L130

Reply:
In that case, I think we should omit it so that we use fewer ways to accomplish the same task (of telling the server not to sign the manifest)